Exec/CMakeLists.txt

```diff
@@ -1,4 +1,2 @@
 add_subdirectory(RegTests)
-if(PELEC_ENABLE_TESTS)
-  add_subdirectory(UnitTests)
-endif()
+#add_subdirectory(UnitTests)
```

Comment:
Did you mean to disable these?

Reply:
Yes, just for CI performance. Since we don't have unit tests anyway, just the framework, we could add them back if we ever write unit tests.